spec/requests/medications_spec.rb

```diff
@@ -1,87 +1,27 @@
 # frozen_string_literal: true
-describe MedicationsController do
-  describe '#print_reminders' do
```

Comment:
Moved to bottom to be consistent with strategies_spec. Being totally new to ruby/rails and rspec, would be curious to understand how a single-method unit test like this fits into request spec model.

Reply:
TLDR answer: It doesn't. I'd delete it.

---

They're duplicate of https://github.com/ifmeorg/ifme/blob/60f8c9b067a9d77d9e1681d954b3346227e47ec1/spec/helpers/reminder_helper_spec.rb#L44-L84

so they're redundant.

----

I actually was confused initially by `#print_reminders` since it was found in `reminder_helper.rb` and was included in the controller via `include ReminderHelper`. 

Looking at the available routes for medications:

![image](https://user-images.githubusercontent.com/7613125/96949487-d8d19680-149c-11eb-8132-2225ceed4a5d.png)

I don't see `#print_reminders` and they're already and should be handled in `reminder_helper_spec.rb` itself. 

Offtopic to the above, usually I see concerns included in models and controllers more than helpers, but I suppose sometimes helpers are occasionally included too, but I see them used for views more.

Source: https://stackoverflow.com/a/18751095/9463774